cmd/src/campaigns_common.go

```diff
@@ -56,6 +57,10 @@ func newCampaignsApplyFlags(flagSet *flag.FlagSet, cacheDir string) *campaignsAp
 		&caf.clearCache, "clear-cache", false,
 		"If true, clears the cache and executes all steps anew.",
 	)
+	flagSet.StringVar(
+		&caf.tempDir, "tmp", tempDir,
+		"Directory for storing temporary data, such as repository archives when executing campaign specs or log files. Default is /tmp. Can also be set with environment variable SRC_CAMPAIGNS_TMP_DIR.",
```

Comment:
The precedence behaviour is implicit right now in the code, but let's document it explicitly.

```suggestion
		"Directory for storing temporary data, such as repository archives when executing campaign specs or log files. Default is /tmp. Can also be set with environment variable SRC_CAMPAIGNS_TMP_DIR; if both are set, this flag will be used and not the environment variable.",
```